docs/configuration_file.md

```diff
@@ -47,106 +47,28 @@ Some options are also mandatory.
 
 * `builder_use_auth` (*optional*, `boolean`) — whether atomic-reactor plugins which in turn use osbs-client from within the build pod should try to authenticate against OpenShift master; defaults to `use_auth`
 
-* `builder_openshift_url` (*optional*, `string`) — url of OpenShift where builder will connect
-
-* `smtp_host` (*optional*, `string`) - SMTP server host, e.g. `smtp.mycompany.com`
-
-* `smtp_from` (*optional*, `string`) - Address to send notifications from, e.g. `user@mycompany.com`
-
-* `smtp_additional_addresses` (*optional*, `string`) - A comma-separated list of additional addresses to include in notifications, e.g. `user1@mycompany.com, user2@mycompany.com`
-
-* `smtp_error_addresses` (*optional*, `string`) - if the plugin has encountered an error the notification will be sent to this list of comma-separated addresses, e.g. `osbs-admin@mycompany.com, osbs-contact@mycompany.com`
-
-* `smtp_email_domain` (*optional*, `string`) - construct email for users if it cannot be guessed from Koji's kerberos principals, e.g. `mycompany.com`
-
-* `smtp_to_submitter` (*optional*, `boolean`) - whether Atomic Reactor should send a notification to koji task submitter
-
-* `smtp_to_pkgowner` (*optional*, `boolean`) - whether Atomic Reactor should send a notification to koji package owner
-
 * `cpu_limit` (*optional*, `string`) — CPU limit to apply to build (for more info, see [documentation for resources](https://github.com/containerbuildsystem/osbs-client/blob/master/docs/resource.md)
 
 * `memory_limit` (*optional*, `string`) — memory limit to apply to build (for more info, see [documentation for resources](https://github.com/containerbuildsystem/osbs-client/blob/master/docs/resource.md)
 
 * `storage_limit` (*optional*, `string`) — storage limit to apply to build (for more info, see [documentation for resources](https://github.com/containerbuildsystem/osbs-client/blob/master/docs/resource.md)
 
-* `reactor_config_secret` (*optional*, `string`) — name of Kubernetes secret holding [atomic-reactor configuration file](https://github.com/containerbuildsystem/atomic-reactor/blob/master/docs/config.md)
-
 * `reactor_config_map` (*optional*, `string`) — name of Kubernetes ConfigMap holding [atomic-reactor configuration file](https://github.com/containerbuildsystem/atomic-reactor/blob/master/docs/config.md)
 
-* `client_config_secret` (*optional*, `string`) — name of Kubernetes secret holding osbs.conf to be used by atomic-reactor in the builder image (this is provided to the orchestrate_build plugin if present)
-
-* `token_secrets` (*optional*, `string`) — whitespace-separated list of secret names with optional mount path in the format secret[:path], which can be used to hold service account tokens referenced by token_file in the osbs_client_secret osbs.conf
-
-* `arrangement_version` (*optional*, `integer`) — default version of inner template to use when creating orchestrator build
```

Comment:
used

Reply:
I'm not sure that arrangement_version other than 6 is valid anymore, but sure.

Reply:
right for now there is only 6 valid, but with new arrangement version it will be still useful